fhir-server-test/src/test/java/testng.xml

```diff
@@ -64,4 +64,9 @@
             <package name="com.ibm.fhir.server.test.terminology" />
         </packages>
     </test>
+    <test name="OperationSpecificTests">
+        <packages>
+            <package name="com.ibm.fhir.server.test.operation" />
+        </packages>
+    </test>
```

Comment:
we have a whole bunch of operation tests under `com.ibm.fhir.server.test`...are you thinking to move those under here?

Reply:
I ended up doing this when I saw the EverythingOperation standalone

Reply:
I do think it makes sense to have a separate package for operation-specific tests, I just think we should move the other ones over as well.

Reply:
I also forgot one key thing with this operation. I wanted it to run last.